app/src/main/java/org/thoughtcrime/securesms/conversation/v2/MessageDetailActivity.kt

```diff
@@ -180,12 +190,20 @@ fun MessageDetails(
                 modifier = Modifier.padding(horizontal = LocalDimensions.current.spacing)
             ) {
                 AndroidView(
-                    factory = {
-                        ViewVisibleMessageContentBinding.inflate(LayoutInflater.from(it)).mainContainerConstraint.apply {
+                    factory = { context ->
+                        // Inflate the view once
+                        ViewVisibleMessageContentBinding.inflate(LayoutInflater.from(context)).mainContainerConstraint
```

Comment:
Although it's existing code I think it's better to use the `rootView` instead of a specific view so you don't accidentally move that view off the root:

```suggestion
                        ViewVisibleMessageContentBinding.inflate(LayoutInflater.from(context)).root
```

Reply:
Changed